### DISCUSSION TITLE
Infinite loop detected

### DISCUSSION THREAD

**Discussion by thelift112** [Jcw87/c2-sans-fight#109] [Q&A]
I'm just wondering, how does it detect infinite loops? Also, is the game sentient?

**Comment by Jcw87** [comment]
No, the game is not sentient. I can assure you that none of our current processor designs will ever become the basis for a sentient computer.

I left a note about how this works here:
https://github.com/Jcw87/c2-sans-fight/issues/39

It doesn't actually know if there is an infinite loop or not. It tries to make a reasonable assumption about it based on how long it has been running for without letting the game loop continue.

**Reply by thelift112** [reply]
> No, the game is not sentient. I can assure you that none of our current processor designs will ever become the basis for a sentient computer.
> 
> I left a note about how this works here: #39
> 
> It doesn't actually know if there is an infinite loop or not. It tries to make a reasonable assumption about it based on how long it has been running for without letting the game loop continue.

I tough it was sentient cause it kept making bones closer to each other but I found a part of the script was accidentally in the loop, and I am needing to force lag to get around the infinite loop system cause I was trying to recreate sine bones.
